internal/control-panel/registry.lua

```diff
@@ -77,6 +77,9 @@ COMMANDS_BY_IDX = {
         params={'--time', '1', '--timeUnits', 'days', '--command', '[', 'fix/stuck-instruments', ']'}},
     {command='fix/stuck-worship', group='bugfix', mode='repeat', default=true,
         params={'--time', '1', '--timeUnits', 'days', '--command', '[', 'fix/stuck-worship', ']'}},
+    {command='fix/zero-noexert-exhaustion', group='bugfix', mode='repeat', default=false,
+        desc='Prevent NOEXERT units (notably Necromancers, Vampires, and Intelligent Undead) from becoming Tired, and fix NOEXERT units which are Tired.',
```

Comment:
Try to match the description length of the other tools. The UI space for displaying this string is limited.

Reply:
Done, shortened description.